BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -184,9 +192,26 @@ extension AttributedLabel {
             urlHandler = environment.urlHandler
             layoutDirection = environment.layoutDirection
 
-            if !isMeasuring, previousAttributedText != attributedText {
-                links = attributedLinks(in: model.attributedText) + detectedDataLinks(in: model.attributedText)
-                accessibilityLinks = accessibilityLinks(for: links, in: model.attributedText)
+            if !isMeasuring {
+                if previousAttributedText != attributedText {
+                    links = attributedLinks(in: model.attributedText) + detectedDataLinks(in: model.attributedText)
+                    accessibilityLinks = accessibilityLinks(for: links, in: model.attributedText)
+                }
+
+                if let shadow = model.shadow {
+                    layer.shadowRadius = shadow.radius
+                    layer.shadowOpacity = Float(shadow.opacity)
+                    layer.shadowOffset = CGSize(width: shadow.offset.horizontal, height: shadow.offset.vertical)
+                    layer.shadowColor = shadow.color.cgColor
+
+                    // enable rasterization on this layer, since text is too complex for a shadow path
+                    layer.shouldRasterize = true
+                    layer.rasterizationScale = layer.contentsScale
```

Comment:
Does this need to be updated if the contents scale changes? Probably very much an edge case 🤔

Reply:
When I tested this the `rasterizationScale` was already correct before being set, so tbh I'm not sure what the circumstances are that you'd need to do this. `BlueprintView` re-renders when it changes windows, so we should pick up scale changes that way. (If there's another way the scale can change, I dunno.)